examples/kotlin/src/test/resources/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
I don't understand why this is in src/test/resources, but I see that it is the same as Java examples.

Reply:
Let me know if you want me to remove this from both of those places.